src/settings-ui/Settings.UI/Views/PowerPreviewPage.xaml

```diff
@@ -35,12 +35,25 @@
                         </controls:Setting.ActionContent>
                     </controls:Setting>
 
-                    <controls:Setting x:Uid="FileExplorerPreview_ToggleSwitch_Preview_Monaco" Icon="&#xE99A;">
-                        <controls:Setting.ActionContent>
-                            <ToggleSwitch IsOn="{x:Bind Mode=TwoWay, Path=ViewModel.MonacoRenderIsEnabled}"
+                    <controls:SettingExpander IsExpanded="True">
+                        <controls:SettingExpander.Header>
+                            <controls:Setting x:Uid="FileExplorerPreview_ToggleSwitch_Preview_Monaco" Icon="&#xE99A;">
```

Comment:
Just an idea: the icon EC7A has the name "DeveloperTools", maybe that'd be appropriate too? :)
https://docs.microsoft.com/en-us/windows/apps/design/style/segoe-fluent-icons-font